HomebrewFormula/survex.rb

```diff
@@ -11,7 +12,21 @@ class Survex < Formula
   depends_on "gettext" => :build
   depends_on "pkg-config" => :build
 
+  head do
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "netpbm" => :build
+    depends_on "gnu-tar" => :build
+  end
+
   def install
+    if build.head?
+      system "autoreconf", "-fiv"
+      system "git", "checkout", "INSTALL"
+      system "curl https://survex.com/software/1.4.2/survex-1.4.2.tar.gz | gtar --strip-components=1 --skip-old-files -zxf -" 
```

Comment:
This will unpack everything from 1.4.2 over the top of the files from git, so will effectively just build 1.4.2.

Reply:
Ah no, because of `--skip-old-files`....